app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from app.cafe import Cafe
```

Comment:
Consider using parentheses for importing multiple classes or modules to maintain consistency with the preferred style.